ci/mkpipeline.py

```diff
@@ -1057,5 +1058,18 @@ def remove_mz_specific_keys(pipeline: Any) -> None:
             )
 
 
+def unparallelize(pipeline: Any) -> None:
+    """Removes parallelism in the test, which will run longer, but exposes some interesting parallelism in some tests."""
```

Comment:
What does "exposes" mean here? That some tests will still run in parallel due to some other kind of parallelization that is not removed here?